aten/src/ATen/core/ivalue.h

```diff
@@ -656,13 +656,10 @@ struct CAFFE2_API WeakIValue final {
 struct StrongTypePtr {
```

Comment:
Can you update the comment for this struct?

Reply:
will do